nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -456,6 +456,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - The initrd ssh daemon module got a new option to add authorized keys via a list of files using `boot.initrd.network.ssh.authorizedKeyFiles`.
 
+- The `redocly-cli` package has been renamed to `redocly`. The old package was aliased for backwards compatibility.
+
```

Comment:
```suggestion
```
if we add an entry to aliases.nix, there is nothing people need to know about and they can fix the warning at some point.